src/config/backpack/base.php

```diff
@@ -58,8 +58,12 @@
         // 'packages/noty/noty.css',
 
         // Load the fonts separately (so that you can replace them at will):
-        'packages/source-sans-pro/source-sans-pro.css',
-        'packages/line-awesome/css/line-awesome.min.css',
+        ['https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400&display=swap', true, [], 'style'],
```

Comment:
This isn't working right now - loading fonts from CDNs like Google Fonts. Why? Because the URL itself doesn't have an extension... so when we save it, THAT doesn't have an extension... 

Even if it did... it would work up to a point. Inside that CSS file, here's what we've got:
```css
@font-face {
  font-family: 'Source Sans Pro';
  font-style: italic;
  font-weight: 400;
  font-display: swap;
  src: url(https://fonts.gstatic.com/s/sourcesanspro/v21/6xK1dSBYKcSV-LCoeQqfX1RYOo3qPa7g.ttf) format('truetype');
}
@font-face {
  font-family: 'Source Sans Pro';
  font-style: normal;
  font-weight: 400;
  font-display: swap;
  src: url(https://fonts.gstatic.com/s/sourcesanspro/v21/6xK3dSBYKcSV-LCoeQqfX1RYOo3aPw.ttf) format('truetype');
}
@font-face {
  font-family: 'Source Sans Pro';
  font-style: normal;
  font-weight: 600;
  font-display: swap;
  src: url(https://fonts.gstatic.com/s/sourcesanspro/v21/6xKydSBYKcSV-LCoeQqfX1RYOo3i54rAkA.ttf) format('truetype');
}
@font-face {
  font-family: 'Source Sans Pro';
  font-style: normal;
  font-weight: 700;
  font-display: swap;
  src: url(https://fonts.gstatic.com/s/sourcesanspro/v21/6xKydSBYKcSV-LCoeQqfX1RYOo3ig4vAkA.ttf) format('truetype');
}
```

Which means we'd be internalizing the CSS file, but NOT the actual font files - those would still be loaded from Google's servers. It would be working. But it wouldn't be GDPR-friendly.

Reply:
See before (v5) and after (v6):
![](https://user-images.githubusercontent.com/1032474/210382831-d14854a3-9aa9-4839-be0b-d37d0998d54c.png)

Reply:
Hmm... there may be a simple solution for the first problem here - see https://github.com/DigitallyHappy/assets/pull/15#discussion_r1060946567